appyters/Clustergrammer_Appyter/README.md

```diff
@@ -0,0 +1,3 @@
+# README
+
+Clustergrammer appyter
```

Comment:
Please expand your README, this should have a long description of what your appyter does.